.github/workflows/regression_test.yml

```diff
@@ -86,19 +92,24 @@ jobs:
             torch-spec: 'torch==2.5.1 --index-url https://download.pytorch.org/whl/cpu'
             gpu-arch-type: "cpu"
             gpu-arch-version: ""
-
-    uses: pytorch/test-infra/.github/workflows/linux_job.yml@main
+            
+    uses: pytorch/test-infra/.github/workflows/linux_job.yml@rocm_experiment
     with:
-      timeout: 120
+      timeout: 60
+      no-sudo: ${{ matrix.gpu-arch-type == 'rocm' }}
+      rocm: ${{ matrix.gpu-arch-type == 'rocm' }}
+      test-infra-ref: rocm_experiment
       runner: ${{ matrix.runs-on }}
       gpu-arch-type: ${{ matrix.gpu-arch-type }}
       gpu-arch-version: ${{ matrix.gpu-arch-version }}
+      docker-image: ${{ matrix.gpu-arch-type == 'rocm' && format('pytorch/manylinux2_28-builder:{0}{1}',
+                                                                      matrix.gpu-arch-type,
+                                                                      matrix.gpu-arch-version)
+                                                           || 'pytorch/almalinux-builder' }}
       script: |
         conda create -n venv python=3.9 -y
         conda activate venv
         echo "::group::Install newer objcopy that supports --set-section-alignment"
-        yum install -y  devtoolset-10-binutils
-        export PATH=/opt/rh/devtoolset-10/root/usr/bin/:$PATH
```

Comment:
The gcctoolset is installed through the dockerfile. Mentioned in this PR: https://github.com/pytorch/pytorch/pull/140157

Reply:
The same is installed for rocm in this PR: https://github.com/pytorch/pytorch/pull/141609